doc/nrf/known_issues.rst

```diff
@@ -224,7 +232,7 @@ NCSDK-16579: Advertising Node Identity and Network ID might not work with the ex
 
   **Workaround:** Don't enable the :kconfig:option:`CONFIG_BT_MESH_ADV_EXT_GATT_SEPARATE` option.
 
```

Comment:
Please add this:

.. rst-class:: v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-1 v1-9-0 v1-8-0 v1-7-1

NCSDK-21780: Sensor types with floating point representation lose precision
  Sensor types with floating point representation lose precision when converted to sensor_value in the sensor API callbacks